packages/client/hmi-client/src/services/charts.ts

```diff
@@ -909,11 +909,12 @@ export function createInterventionChartMarkers(interventions: Intervention[]): a
 	return [markerSpec, labelSpec];
 }
 
-export function createInterventionChart(interventions: Intervention[]) {
+export function createInterventionChart(interventions: Intervention[], chartOptions: BaseChartOptions) {
```

Comment:
`createInterventionChart` should support the fields as defined in `BaseChartOptions` (namely title and legend), or conversely use its own options typing definitions and not piggy back. This mix and match and partial support makes it hard to know exactly which features/options are available.

Reply:
sure, Im supporting `title` now and doing something similar to the error chart where I am omitting `legend` from the `BaseChartOptions` type